xtext-website/community.html

```diff
@@ -558,6 +566,14 @@ <h2>What others have built with Xtext</h2>
 						<td>Michel Simeon</td>
 					</tr>
 
+					<tr>
+						<td><a href="https://github.com/xtext/maven-xtext-example">Maven Xtext Example</a></td>
+						<td>A small example to show how to configure a Maven build for an Xtext language and how to use it from Maven and Gradle.</td>
```

Comment:
@cdietrich : Could you please review the description of the [maven-xtext-example](https://github.com/xtext/maven-xtext-example) project? Any feedback is highly apprechiated!